CHANGELOG.md

```diff
@@ -23,6 +23,10 @@
 * [BUGFIX] Fix `MimirCompactorHasNotUploadedBlocks` alert false positive when Mimir is deployed in monolithic mode. #1901
 * [BUGFIX] Do not trigger `MimirAllocatingTooMuchMemory` alerts if no container limits are supplied. #1905
 
+### Jsonnet
+
+* [CHANGE] Remove use of `querier.query-store-after` and `blocks-storage.bucket-store.ignore-blocks-within` CLI flags since the values now match defaults. #1915
```

Comment:
By convention we prefix every CLI flag with `-` to signal more clearly it's a CLI flag.

```suggestion
* [CHANGE] Remove use of `-querier.query-store-after` and `-blocks-storage.bucket-store.ignore-blocks-within` CLI flags since the values now match defaults. #1915
```